app/portainer/components/datatables/components/TableHeaderRow.tsx

```diff
@@ -26,6 +26,9 @@ export function TableHeaderRow<
           headerProps={{
             ...column.getHeaderProps({
               className: column.className,
+              style: {
+                width: (column.id === 'selection' || column.id === 'actions') ? '5px' : '',
```

Comment:
what if we have many actions, so we want more then 5px?

Reply:
changed it from '5px' to column.width, that way whats seems to happen is that the columns get a fixed size and don't change if I alter the shown columns (which is pretty much what I intended)

Reply:
(resolving this thread, continue discussion below)